packages/features/data-table/components/filters/DateRangeFilter.tsx

```diff
@@ -84,7 +88,11 @@ export const DateRangeFilter = ({ column }: DateRangeFilterProps) => {
   const [endDate, setEndDate] = useState<Dayjs | undefined>(
     filterValue?.data.endDate ? dayjs(filterValue.data.endDate) : getDefaultEndDate()
   );
-  const [selectedPreset, setSelectedPreset] = useState<PresetOption>(DEFAULT_PRESET);
+  const [selectedPreset, setSelectedPreset] = useState<PresetOption>(
```

Comment:
In case `filterValue` is filled by url params from the first rendering, we use it instead of DEFAULT_PRESET.